pkg/tfgen/docs.go

```diff
@@ -1174,6 +1175,43 @@ func (g *Generator) convertExamples(docs, name string, stripSubsectionsWithError
 	return output.String()
 }
 
+// A ConversionError occurs when convert.Convert yields a panic.
+// This can be removed when https://github.com/pulumi/pulumi-terraform-bridge/issues/477
+// is resolved. ConversionError exposes the stacktrace of the panic so callers
+// can choose to pass the trace along to the user or swallow it.
+type ConversionError struct {
+	// panicArg is the argument that was passed to panic() during conversion.
+	panicArg interface{}
+	// trace is the captured stacktrace.
+	trace string
+	// wrappedErr is the error message provided by this struct.
+	wrappedErr error
+}
+
+// construct a new ConversionError. The argument is expected to be
+// the value that was recovered from the panic.
+func newConversionError(panicArg interface{}, trace string) *ConversionError {
+	var err = fmt.Errorf("panic converting HCL: %s", panicArg)
+	return &ConversionError{
+		panicArg:   panicArg,
+		trace:      trace,
+		wrappedErr: err,
+	}
+}
+
+// StackTrace returns the stacktrace of the error.
+func (err *ConversionError) StackTrace() string {
+	return err.trace
+}
+
+// Return the err-representation of this struct.
+func (err *ConversionError) Error() string {
```

Comment:
Nit - should we add `Unwrap` as well to support unwrapping and `errors.As` and `errors.Is`?

Reply:
TIL about `Unwrap`!